File: foundation/foundation 2 month/p_exam/11-10-24/v2/ex_1.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
#include <limits.h>
void check(int ms[],int k)
{
	puts("");
	int max=INT_MIN;
	int min=INT_MAX;
	for(int i=0;i<k;i++)
	{
		if(ms[i]>max)
		max=ms[i];
		else if(ms[i]<min)
		min=ms[i];
	}
	printf("Min=%d\n",min);
	printf("Max=%d\n",max);
	printf("Ayilmasi: %d",max-min);
}
int enter()
{
	return rand()%100;
}

int main()
{
	srand(time(0));
	int n;
	printf("N ni kiriting: ");
	scanf("%d",&n);
	int arr[n];
	for(int i=0;i<n;i++)
	{
		arr[i]=enter();
		printf("%4d",arr[i]);
	}
	check(arr,n);
}

File: full stack/full stack 1 month/lesson6/classwork/ex_1.js
arr = [1,2,3,1,5,5,5,4,5,7,5,3]
ms ={}
for (let i =0;i<arr.length;i++)
{
    let s = arr[i]
    if(s in ms)
    {
        ms[s]+=1
    }
    else
    {
        ms[s]=1
    }
}
console.log(ms)
 let a = Object.keys(ms).length; 

let max = 0
var s;
let min = 123
let cnt=0
var d;
for (let i = 1; i<a;i++)
{
    if (ms[i]>max)
    {
        max = ms[i]
        s = i
    }
    else if(ms[i]< min)
    {
        min =ms[i]
        d = i
    }
}
console.log(s,max)
console.log(d,min)